### DISCUSSION TITLE
[交流灌水]我都不知道从哪里开始笑了

### DISCUSSION THREAD

**Discussion by youyiMC** [share121/inter-knot#2983] [其他]
登上GitHub对于反米小学生就是一大难题了吧。说不定其中还有人自认为爆破成功了<img src="https://img-static.mihoyo.com/communityweb/upload/460474087917ad69a4ff975a3132a0df.png" alt="妮可 哈哈" width="75" >
![Screenshot_2024-08-19-09-21-57-520_tv danmaku bili-edit](https://github.com/user-attachments/assets/5e4eb339-4af7-4faf-856f-615e3aa3a17e)

**Comment by Elysia-Ego-093** [comment]
坐等米黑无能狂怒

**Comment by bangbu01** [comment]
堪比暗网的保密程度可不是吹的

**Reply by ZenlessZoneZero01** [reply]
那倒不至于

**Reply by lantainAirblue** [reply]
这点我可以以为的绳网声望担保比深潜还是差远了。不过可以了已经。
![Uploading 2018-07-02 210435.jpg…]()

**Comment by LinkboyTvT** [comment]
要电脑，要油猴，要GitHub，不开加速器估计注册都是问题，跟何况大大命令帖子规范<img src="https://img-static.mihoyo.com/communityweb/upload/460474087917ad69a4ff975a3132a0df.png" alt="妮可 哈哈" width="75" >

**Reply by mzyxz0410** [reply]
其实不一定要电脑，手机也可以，不过对于那些人来说已经够难了

**Reply by youyiMC** [reply]
其实有办法裸连GitHub的，但以反米魔怔人的德行估计没兴趣研究╭☞( ￣ ▽￣)╭☞

**Reply by MichaelJason772** [reply]
我是手机下的，当时废了我一个多小时对于一个从来没有接触梯子的我来说就已经是一个挑战了，更别说当时只有网页版，恐怕小学生网页都找不到，下载都不下载不来

**Reply by Robin-txdy** [reply]
的确是难，我当时弄了好久

**Reply by mengsiliulian** [reply]
我说实话，我直接用手机就搞好了（虽然用起来挺麻烦）

**Comment by lllliufeng** [comment]
举报5次自动隐藏，还是相当不错的

**Comment by fashibuding1214** [comment]
真不知道怎么形容这些魔怔人好→_→

**Reply by 529481** [reply]
小混混加小流氓是这样的

**Comment by DDtoma** [comment]
有相当一部分是拿钱办事，还有一群傻的。

**Comment by LT-UT** [comment]
希望up能做一些反制手段<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Comment by 502awa** [comment]
吃瓜<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" ><img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by isle200inf** [comment]
这个是真魔怔，下面评论都骂他，而且，爆github，爆笑了

**Reply by Arocohol** [reply]
可以搬来让大伙看看嘛（）

**Reply by Robin-txdy** [reply]
> 可以搬来让大伙看看嘛（）

可以去贴吧搜他的名字，趁现在帖子还在，快去看乐子

**Comment by 1ctrl-cv** [comment]
看乐了

**Comment by Eufo0305** [comment]
米黑的智商还不够格。

**Comment by warning-Kirov-reporting** [comment]
有没有下面的评论截图，想看<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by Robin-txdy** [reply]
直接去贴吧搜名字就好了，楼主截图上有

**Comment by Lyfer123** [comment]
![IMG_5580](https://github.com/user-attachments/assets/fdb14322-91b6-444b-896a-b7c8a41ac959)

**Reply by RaInWH** [reply]
捉到一只486（）

**Comment by 00zzxx** [comment]
不是，爆github？真的假的？

**Comment by Junjun280** [comment]
哈哈哈哈哈哈哈，笑死了，乐![image](https://github.com/user-attachments/assets/d8303464-9468-41ae-b15c-15ef8f7ba80d)

**Comment by yinlang520** [comment]
![68747470733a2f2f75706c6f61642d6262732e6d69796f757368652e636f6d2f75706c6f61642f323032332f30332f30392f33313733626333653639666366656265636664613734313334306339616432325f363834313235303331303735363237353](https://github.com/user-attachments/assets/802098f2-9b15-4ea7-bef6-b9b667e17690)

**Comment by chuibada** [comment]
笑死

**Comment by Arocohol** [comment]
搬石，爱看（<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >）

**Comment by arlcius** [comment]
![c64ac6c88f88054abf5111b8123a88e1](https://github.com/user-attachments/assets/45a26bd3-7b65-44ea-9eaa-33cf77627e32)

**Comment by EP-office** [comment]
啊？<img src="https://bbs-static.miyoushe.com/static/2023/04/28/f2f1488ffbfff3fa96d4db6047e5f409_6776930953450600054.png" alt="米仔_不了" width="75" >

**Comment by fqing8485** [comment]
看乐子了

**Comment by WinterSamu** [comment]
话说这些米默怎么跟蟑螂一样到哪哪出现呢？<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3d389fd2885b008e46d843c552c02bf3_5073438697399152551.png" alt="开拓者 吃瓜脸" width="75" >

**Reply by bushigemengsishuihaiyinga** [reply]
赛博生命是这样的

**Comment by shengwangyonghu** [comment]
乐了哈哈哈哈哈哈

**Comment by ysdtal** [comment]
他们先看懂了怎么玩发帖子再说吧，别等会又被打成成仙帆船的了。

**Comment by For-elysia** [comment]
其实我一直以为2024年了，只有人机对冲锋感兴趣，看来是信息茧房太深了

**Comment by 3566166260** [comment]
乐

**Comment by Robin-txdy** [comment]
有哪位朋友有截图，想看乐子

**Reply by Robin-txdy** [reply]
刚看完乐子回来，笑了半个小时

**Comment by nzj2004** [comment]
乐了，不过之前不是因为一个bug，被删了数据吗？
